apps/backend/src/datasources/postgres/records.ts

```diff
@@ -353,7 +353,7 @@ export interface InstitutionRecord {
   readonly institution_id: number;
   readonly institution: string;
   readonly country_id: number;
-  readonly verified: boolean;
+  readonly ror_id?: string;
```

Comment:
Why the `ror_id` is optional? Due to the database patch, the column will be added to the Institution table, isn't it?